app/src/pages/inside/stepPage/modals/makeDecisionModal/makeDecisionTabs/makeDecisionTabs.jsx

```diff
@@ -0,0 +1,179 @@
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import { DottedPreloader } from 'components/preloaders/dottedPreloader';
+import { useState } from 'react';
+import { uniqueId } from 'common/utils';
+import { useIntl } from 'react-intl';
+import {
+  COPY_FROM_HISTORY_LINE,
+  MACHINE_LEARNING_SUGGESTIONS,
+  SELECT_DEFECT_MANUALLY,
+} from '../constants';
+import { messages } from '../messages';
+import styles from './makeDecisionTabs.scss';
+
+const cx = classNames.bind(styles);
+
+export const MakeDecisionTabs = ({
+  tabs,
+  toggleTab,
+  suggestedItems,
+  loadingMLSuggest,
+  modalState,
+  setModalState,
+  isBulkOperation,
+  isAnalyzerAvailable,
+}) => {
+  const { formatMessage } = useIntl();
+  const [selectedMLSuggest, setSelectedMLSuggest] = useState(null);
+  const [animationSuggest, setAimationSuggest] = useState(true);
+  const suggestVariant = ['one', 'two', 'three'];
+  const [selectDefectTab, machineLeariningTab, copyFromHistoryLineTab] = tabs;
+  const source = modalState.suggestChoice;
+
+  const selectMachineLearningSuggestionItem = (index, itemId) => {
+    if (itemId && itemId === source.id && machineLeariningTab.isOpen) {
+      return;
+    }
+    toggleTab(MACHINE_LEARNING_SUGGESTIONS);
+    setSelectedMLSuggest(index);
+
+    if (index === 0) {
+      setAimationSuggest(false);
+    }
+
+    const { testItemResource, logs, suggestRs } = suggestedItems.find(
+      (item) => item.testItemResource.id === itemId,
+    );
+    setModalState({
+      ...modalState,
+      decisionType: MACHINE_LEARNING_SUGGESTIONS,
+      issueActionType: '',
+      suggestChoice: { ...testItemResource, logs, suggestRs },
+    });
+  };
+
+  const selectSideItem = (tab) => {
+    toggleTab(tab);
+
+    setModalState({
+      ...modalState,
+      decisionType: tab,
+    });
+  };
+  return (
+    <div className={cx('make-decision-tabs')}>
+      {!isBulkOperation && (
+        <div className={cx('tabs')}>
+          <div
+            onClick={() => selectSideItem(SELECT_DEFECT_MANUALLY)}
+            className={cx('side-block', {
+              'side-block-active': selectDefectTab.isOpen,
+            })}
+          >
+            <p className={cx('side-block-title')}>{formatMessage(messages.manual)}</p>
+            <p className={cx('side-block-text')}>{formatMessage(messages.selection)}</p>
+          </div>
+          <div className={cx('central-block')}>
+            {loadingMLSuggest && (
```

Comment:
Seggests should be shown for bulk operation in case of suggests was requested for the cluster (according to EPMRPP-68410)